index.js

```diff
@@ -1,64 +1,78 @@
-let idCounter = localStorage.getItem('counter')
-  ? Number(localStorage.getItem('counter'))
-  : localStorage.setItem('counter', 0);
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author, id = null) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
```

Comment:
- [optional] Please kindly ensure that the id value is updated in the local storage if book(s) are removed, this should have been done in the previous milestone according to the requirement. 👍 

## Your localStorage 

<img width="277" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/226569976-ff6b6b3c-a494-49f8-8443-d2bd3ccd384c.png">

Reply:
We made it incremental so that it doesn't conflict and is always unique.

Reply:
- [ OPTIONAL] It's very creative of your to use this approach. But please note that the requirements stated that previous functionality should be maintained.

![reqs](https://user-images.githubusercontent.com/99656687/226584954-5c1fdacb-fa15-484f-9e91-3ba781744c3f.png)

Reply:
@wayungi , thanks for bring this out that's a mistake on my part. @mahabubx7 Please kindly take note that the optional change should have been mandatory as @wayungi pointed out.

Reply:
> We made it incremental so that it doesn't conflict and is always unique.

@mahabubx7, yes you're right! I only suggest that when a book is removed from the app, the id values of the remaining books should be updated and be serially ordered. 👍